src/i18n/z_language.erl

```diff
@@ -149,7 +149,7 @@ properties(Code, Data) ->
 
 
 %% @doc Sorts a properties list.
--spec sort_properties(List::list(), SortKey::string()) -> list().
+-spec sort_properties(List :: list(), SortKey :: atom()) -> list().
 sort_properties(List, SortKey) ->
     lists:sort(fun({_, PropsA}, {_, PropsB}) ->
         z_string:to_lower(proplists:get_value(SortKey, PropsA)) =< z_string:to_lower(proplists:get_value(SortKey, PropsB))
```

Comment:
Line 155 is too long:         z_string:to_lower(proplists:get_value(SortKey, PropsA)) =< z_string:to_lower(proplists:get_value(SortKey, PropsB)).